src/react-components/src/components/00-base/Container/Modal.tsx

```diff
@@ -0,0 +1,31 @@
+/* Full-screen modal that appears on top of the body */
+import * as React from "react";
+import bem from "../../../utils/bem";
+
+export interface ModalProps {
+  /** Used by BEM CSS */
```

Comment:
Since these comments are probably going to be repeated across every component ever, what do you think of these descriptors:

```
/** Optional modifiers array for use with BEM. See how to work with modifiers and BEM here: http://getbem.com/introduction/ */
modifiers?: string[];

/** Optional blockName for use with BEM. See how to work with blockNames and BEM here: http://getbem.com/introduction/ */
modifiers?: string[];
```

I actually don't love that but it's a start... ideally #109 would exist and we would point them there

Reply:
It might be really long to show up in the PropTypes thing in storybook ... Maybe something that's good enough get people googling in the right direction?  
Maybe "Modifiers/Blockname for use with BEM" ? (Optional is already inferred from the type)

Reply:
The PropTypes thing shows up as a handy little table—I think it's okay for them to be wordy. We can leave them short if you're prefer though. (And as for the optional thing omg I wrote optional all over the Hero descriptors, I just am a terrible technical writer bleh)